lisp/forge-github.el

```diff
@@ -225,8 +225,8 @@
         (when bump
           (forge--set-id-slot repo pullreq 'assignees .assignees)
           (forge--set-id-slot repo pullreq 'review-requests
-                              (--map (cdr (cadr it))
-                                     (car .reviewRequests)))
+                              (--map (cdr (cadr (car it)))
+                                     .reviewRequests))
```

Comment:
When I saw that `car` I also thought "ah!! easy fix", but ... *now I got it*:

https://github.com/stellargraph/stellargraph/pull/738 actually only has one review request. One *outstanding* review requests and two (already performed) *reviews*.

Okay then! In that case please fix this bug in one commit and use another commit to add the new feature below.

Reply:
Oh, and please point me to a pull-request that currently has two open review requests so that I can verify myself that this fix really works.

Reply:
https://github.com/kubernetes/kubernetes/pull/88249